posts/2015-04-10-symfony-live-2015.md

```diff
@@ -0,0 +1,214 @@
+La septième édition du SymfonyLive Paris s'est tenue le jeudi 9 avril à la Cité Internationale Universitaire de Paris. Retour sur cet évènement.
+
+## 10 ans déjà... quid de Symfony 3.0 ? par Fabien Potencier
+
+Tout en décontraction, Fabien Potencier nous avoue qu'il n'est pas très "tech" ni très connecté. Il a découvert Whatsapp il y a 6 mois grâce à sa... mère.
+
+Petite anecdote, le premier site développé avec Symfony était un site marchand de lingerie. Le client n'a pas payé et c'est pourquoi il a été décidé de livrer Symfony à la communauté Open-Source. A la base, le framework s'appelait "Sensio Framework" et le "f" de Symfony vient de "Framework".
+
+Fabien nous parle de la responsabilité du développement de Symfony vis à vis de ces utilisateurs, les versions LTS, la roadmap... Le passage de la version 1.4 à la version 2 a été douloureuse mais nécessaire. Symfony 3 ne sera pas une révolution mais une évolution. Exemple avec le composant Asset qui a été recodé complètement. L'ancien et le nouveau cohabitent sur la version 2.7 pour laisser le temps aux applications de migrer. C'est la philosophie actuelle du développement de Symfony : préserver la rétro-compatibilité. La version 3 sera une version déduite des composants dépréciés. Symfony 3 est en réalité déjà prête car toutes les fonctionnalités sont dans la 2.7. Il ne reste donc plus qu'à "nettoyer" les composants dépréciés. Il y aura une version 2.8 qui sortira en même temps que la 3.0 et qui contiendra les hotfix et refactoring réalisés sur la version 3. La 2.8 sera également LTS.
+
+Fabien nous avoue que ce qu'il voulait faire avec le micro-framework Silex, Laravel l'a fait, et a bien largement trouvé sa communauté. Silex is dead ?
```

Comment:
* Pas d'espace avant un `?` en anglais. -> Silex is dead?
* D'ailleurs j'aurais plutôt écris `Is Silex dead?`